docs/docs/databases/docker-add-drivers.mdx

```diff
@@ -8,86 +8,75 @@ version: 1
 ## Adding New Database Drivers in Docker
 
 Superset requires a Python database driver to be installed for each additional type of database you
-want to connect to. When setting up Superset locally via `docker compose`, the drivers and packages
-contained in
-[requirements.txt](https://github.com/apache/superset/blob/master/requirements.txt) and
-[requirements-dev.txt](https://github.com/apache/superset/blob/master/requirements-dev.txt)
-will be installed automatically.
+want to connect to.
 
-In this section, we'll walk through how to install the MySQL connector library. The connector
-library installation process is the same for all additional libraries and we'll end this section
-with the recommended connector library for each database.
+In this example, we'll walk through how to install the MySQL connector library. The connector
+library installation process is the same for all additional libraries.
```

Comment:
@sfirke any reason we're force breaking lines in markdown? I'm seeing this throughout the docs and it seems somewhat atypically for us to be doing this given that markdown auto-wraps.

Reply:
Nope just ignorance on my part.  What is the guidance for breaking lines - should I manual newline after X (?) characters outside of markdown and then inside markdown, have indefinitely-long lines?

Reply:
And to be clear, is all of this documentation markdown - in which case do I do no manual line breaks in my IDE?

If there is a standard, maybe we can add it to the contributing guidelines (I will confess it could be there and I missed it).

Reply:
Forgot to do the @ mention @john-bodley  ^^

Reply:
@sfirke I gather the logic should only be to include a manual newline when necessary from a rendering perspective,  new paragraph, bullet list, etc. In your editor enabling "word wrap" should ensure that you're not dealing with any horizontal scrolling.

I wonder if `prettier` or similar could somehow ensure that our Markdown was rendered correctly.

Reply:
Thanks @john-bodley, I enabled that in my VSCode and it's a big help.  I pushed changes that undid my manual newlines.

Reply:
This came up here https://github.com/apache/superset/pull/26923/files where i was pointed to this comment. Wrote this over there:
> Personally I like line breaks and happily will get into a tabs-vs-space religious debate over it :) Main point is around "diffing". If you have a long paragraph and fix a typo, you're in the git blame for that whole line, and diffs will be much more verbose than needed. I vote for line breaks soft at 80-90, hard at 100, and across file most file types.

Reply:
I've added this item to the town hall agenda for the next meeting.